### PR TITLE
Add v4 TPU manifests samples

### PR DESCRIPTION
**Pull request by richardsliu**
Updates TPU v4 sample manifests for Kuberay v1.1

### CODE REVIEW

File: ray-operator/config/samples/ray-cluster-tpu-v4-singlehost.yaml [hunk 1]
```diff
@@ -1,116 +1,125 @@
-# This template contains a Kuberay cluster using a 2x2x2 TPU v4 PodSlice.
```
Comment (kevin85421) [inline ray-operator/config/samples/ray-cluster-tpu-v4-singlehost.yaml:1]: I have similar comments as `ray-cluster-tpu-v4-multihost.yaml`.

**Review by kevin85421 — COMMENTED** [review]
Would you mind renaming the file name from `ray-cluster-tpu-v4-multihost.yaml` to `ray-cluster.tpu-v4-multihost.yaml` and `ray-cluster-tpu-v4-singlehost.yaml` to `ray-cluster.tpu-v4-singlehost.yaml`

File: ray-operator/config/samples/ray-cluster-tpu-v4-multihost.yaml [hunk 2]
```diff
@@ -0,0 +1,129 @@
+apiVersion: ray.io/v1
+kind: RayCluster
+metadata:
+  labels:
```
[the remark this answers is not in the document]
Comment (richardsliu) [reply ray-operator/config/samples/ray-cluster-tpu-v4-multihost.yaml]: Removed.

File: ray-operator/config/samples/ray-cluster-tpu-v4-multihost.yaml [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+          {}
+        tolerations:
+          []
+        nodeSelector:
+          cloud.google.com/gke-tpu-accelerator: tpu-v4-podslice
+          cloud.google.com/gke-tpu-topology: 2x2x2
+      metadata:
+        annotations:
+          key: value
+        labels:
```
[the remark this answers is not in the document]
Comment (richardsliu) [reply ray-operator/config/samples/ray-cluster-tpu-v4-multihost.yaml:104]: I kept some of the ones needed for telemetry.

**Comment by richardsliu** [comment]
@kevin85421 All done, please take a look.

**Review by kevin85421 — COMMENTED** [review]
Would you mind fixing the CI errors related to RayCluster? You can run the tests locally by following https://github.com/ray-project/kuberay/blob/master/ray-operator/DEVELOPMENT.md#running-configuration-tests-locally.

**Comment by richardsliu** [comment]
Fixed the broken tests.

**Review by kevin85421 — APPROVED** [review]

**Pull request merged**